addons/ui/README.md

```diff
@@ -9,3 +9,4 @@ Changes the chat contrast on the map to allow easier reading.
 The people responsible for merging changes to this component or answering potential questions.
 
 - [VKing](https://github.com/VKing6)
+- [Jonpas](https://github.com/jonpas)
```

Comment:
Damn, that is left over from the bigger PR in which I am waiting for new commands that didn't make it into 1.50, I just cherry-picked. :)